### ISSUE TITLE
lavPredict if level = 2

### ISSUE THREAD

**Issue by BernhardMeier** [yrosseel/lavaan#108]
Hey,

I get the following error when I try to get the fb values for the multilevel demo model:

```
model <- '
  level: 1
    fw =~ y1 + y2 + y3
  level: 2
    fb =~ y1 + y2 + y3
    fb ~ w1 
    y3 ~ w1
'

fit <- sem(model = model, data = Demo.twolevel, cluster = "cluster")
lavPredict(fit, level=2)

```

Error message: **Error in dimnames(x) <- dn : 
  length of 'dimnames' [2] not equal to array extent**

It seems it has something to do with the first level. If I specify  "fw ~ x1 + x2 + x3" like in the tutorial it works 

```
model <- '
  level: 1
    fw =~ y1 + y2 + y3
    fw ~ x1 + x2 + x3
  level: 2
    fb =~ y1 + y2 + y3
    fb ~ w1 
    y3 ~ w1
'

fit <- sem(model = model, data = Demo.twolevel, cluster = "cluster")
lavPredict(fit, level=2)
```

**Comment by BernhardMeier** [comment]
Okay, so if you set label=F it works:

```
model <- '
  level: 1
    fw =~ y1 + y2 + y3
  level: 2
    fb =~ y1 + y2 + y3
    fb ~ w1 
    y3 ~ w1
'

fit <- sem(model = model, data = Demo.twolevel, cluster = "cluster")
lavPredict(fit, level=2, label=F)
```

The output looks strange though

**Comment by yrosseel** [comment]
Thanks for the report. This is now fixed in dev lavaan 0.6-2.1279.

Yves.
